tools/migrate_data_to_c

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
```

Comment:
I think we can remove this file in favour of the equivalent script Brotenko made.
Also this needs a `.py` extension if we're keeping it

Reply:
yeah I vote remove since brotenko's doesn't have the ordering issue

Reply:
I third this movement